src/lib/runAlloy.js

```diff
@@ -2242,9 +2242,9 @@ governing permissions and limitations under the License.
         return function (config) {
           var componentRegistry = createComponentRegistry();
           var imsOrgId = config.imsOrgId,
-            propertyId = config.propertyId,
+            configId = config.configId,
             cookieDomain = config.cookieDomain;
-          var cookieName = ALLOY_COOKIE_NAME + "_" + propertyId;
+          var cookieName = ALLOY_COOKIE_NAME + "_" + configId;
```

Comment:
Hey @Aaronius, I think we need to update this to use the new Cookie naming structure?

Reply:
The contents of `runAlloy.js` is currently manually copied in from the alloy build. In other words, you should run `npm run build:prod` in Alloy, then copy the contents of `dist/reactor/alloy.js` from Alloy and paste it into this `src/lib/runAlloy.js` file (see the comments for where to paste it). Is this answering your question?